packages/grafana-ui/src/components/ValueMappingsEditor/DraggableMappingRow.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { DraggableProvidedDragHandleProps } from 'react-beautiful-dnd';
+import { GrafanaTheme } from '@grafana/data';
+import { useStyles } from '../../themes';
+import { cx, css } from 'emotion';
+
+interface DraggableMappingRowProps extends DraggableProvidedDragHandleProps {
+  label?: string;
+}
+
+export const DraggableMappingRow = React.forwardRef<HTMLDivElement, DraggableMappingRowProps>(
+  ({ children, label, tabIndex, ...otherProps }, ref) => {
+    const styles = useStyles(getDraggableRowStyles);
+    return (
+      <>
+        <div className="gf-form-label gf-form-label--justify-left width-5" ref={ref} {...otherProps}>
+          <i className={cx('fa fa-ellipsis-v', styles.draggable)} />
+          {label}
+        </div>
+        {children}
+      </>
+    );
+  }
+);
+const getDraggableRowStyles = (theme?: GrafanaTheme) => {
+  if (!theme) {
+    theme = {} as GrafanaTheme;
+  }
```

Comment:
@dprokop -- obviously something funky here.  this was required for typings

`useTheme()` makes an optional theme?

Reply:
uhm, must have missed that. No need for that at all, useTheme assures theme object.